lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,26 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules"
```

Comment:
@justin808 What do you think about adding an exception in the linter task. With rescue LoadError nil, the task doesn't get loaded if the gem doesn't exist and the precompile doesn't throw an error. Tested with rake -T and rake assets:precompile, the task gets completely ignored.

If you want to go with your plan, should we let the user add the gem or have the generator add it.